tests/node/test_get_info.py

```diff
@@ -0,0 +1,138 @@
+"""
+Provide tests for command line interface's node get information command.
+"""
+import json
+
+import pytest
+from click.testing import CliRunner
+
+from cli.constants import (
+    FAILED_EXIT_FROM_COMMAND_CODE,
+    NODE_27_IN_TESTNET_ADDRESS,
+    PASSED_EXIT_FROM_COMMAND_CODE,
+)
+from cli.entrypoint import cli
+from cli.utils import dict_to_pretty_json
+
+
+def test_get_node_info():
+    """
+    Case: get node information.
+    Expect: flag is synced and peer count are returned.
+    """
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'node',
+        'get-info',
+        '--node-url',
+        NODE_27_IN_TESTNET_ADDRESS,
+    ])
+
+    node_information = json.loads(result.output).get('result').get('information')
+
+    node_is_synced = node_information.get('is_synced')
+    node_peer_count = node_information.get('peer_count')
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert isinstance(node_is_synced, bool)
+    assert isinstance(node_peer_count, int)
+
+
+def test_get_node_info_without_node_url(mocker, node_information):
+    """
+    Case: get node information without passing node URL.
+    Expect:flag is synced and peer count are returned from node on localhost.
```

Comment:
`flag is synced and peer count are returned from node on localhost `>` the flag is synced and peer count are returned from a node on localhost`

Reply:
@Alladin9393, done.